webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -166,6 +161,15 @@ def test_home(self):
             ok_(link['url'] in response.content)
             ok_(link['label'] in response.content)
 
+        # The access should disappar if you cease to be active
+        User.objects.filter(id=user.id).update(is_active=False)
```

Comment:
Glad you asked. The reason is because the `_login` method works like this:
```python
user = User.objects.create_user(...)
# at this point, the user.last_login is None
self.client.login(username=user.username, ...)
# it's now the same user from above. The self.client.login() updates the user's last_login
return user
```
So even though the `_login` method does both a user creation AND a login, the user it returns is one that doesn't have a `last_login` set.